test/types/records/init/copy/copyWithRefOther.chpl

```diff
@@ -1,15 +1,20 @@
 record R {
   var x: int;
-
+  
```

Comment:
Looks like you added a tab accidentally here

Reply:
Yep!